packages/roosterjs-content-model/lib/domToModel/processors/childProcessor.ts

```diff
@@ -17,6 +17,10 @@ export const childProcessor: ElementProcessor<ParentNode> = (
     const [nodeStartOffset, nodeEndOffset] = getRegularSelectionOffsets(context, parent);
     let index = 0;
 
+    if (context.shouldSkipProcessElement?.child?.(parent, context)) {
+        return;
+    }
```

Comment:
Actually we can't just skip delimiter, because there may be selection at the delimiter, then we still need to handle selection.

We can let knownElementProcessor to keep process it, but if we see it is a delimiter, just don't create any model for it.

Reply:
Added Delimiter processor to check the selection as suggested